Please resolve the remaining lint failures:

```
make[2]: Entering directory '/home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/exporter/elasticexporter'
Check License finished successfully
running misspell -error
golangci-lint run --allow-parallel-runners
Error: exporter_test.go:39:6: shadow: declaration of "err" shadows declaration at line 36 (govet)
		if err := tt.Shutdown(context.Background()); err != nil {
		   ^
Error: exporter_test.go:71:6: shadow: declaration of "err" shadows declaration at line 68 (govet)
		if err := tt.Shutdown(context.Background()); err != nil {
		   ^
Error: exporter_test.go:98:6: shadow: declaration of "err" shadows declaration at line [95](https://github.com/open-telemetry/opentelemetry-collector-contrib/runs/6609874669?check_suite_focus=true#step:6:96) (govet)
		if err := tt.Shutdown(context.Background()); err != nil {
		   ^
make[2]: Leaving directory '/home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/exporter/elasticexporter'
make[2]: *** [../../Makefile.Common:[105](https://github.com/open-telemetry/opentelemetry-collector-contrib/runs/6609874669?check_suite_focus=true#step:6:106): lint] Error 1
make[1]: *** [Makefile:180: exporter/elasticexporter] Error 2
```

--- a/exporter/elasticexporter/exporter_test.go
+++ b/exporter/elasticexporter/exporter_test.go
@@ -36,7 +36,7 @@ func TestTracesExporter(t *testing.T) {
 	tt, err := obsreporttest.SetupTelemetry()
 	require.NoError(t, err)
 	defer func() {
-		if err := tt.Shutdown(context.Background()); err != nil {
+		if err = tt.Shutdown(context.Background()); err != nil {
 			t.Fatalf("failed to stop: %v", err)
 		}
 	}()
@@ -68,7 +68,7 @@ func TestMetricsExporter(t *testing.T) {
 	tt, err := obsreporttest.SetupTelemetry()
 	require.NoError(t, err)
 	defer func() {
-		if err := tt.Shutdown(context.Background()); err != nil {
+		if err = tt.Shutdown(context.Background()); err != nil {
 			t.Fatalf("failed to stop: %v", err)
 		}
 	}()
@@ -95,7 +95,7 @@ func TestMetricsExporterSendError(t *testing.T) {
 	tt, err := obsreporttest.SetupTelemetry()
 	require.NoError(t, err)
 	defer func() {
-		if err := tt.Shutdown(context.Background()); err != nil {
+		if err = tt.Shutdown(context.Background()); err != nil {
 			t.Fatalf("failed to stop: %v", err)
 		}
 	}()

--- a/exporter/elasticexporter/exporter_test.go
+++ b/exporter/elasticexporter/exporter_test.go
@@ -36,9 +36,7 @@ func TestTracesExporter(t *testing.T) {
 	tt, err := obsreporttest.SetupTelemetry()
 	require.NoError(t, err)
 	defer func() {
-		if err = tt.Shutdown(context.Background()); err != nil {
-			t.Fatalf("failed to stop: %v", err)
-		}
+		require.NoError(t, tt.Shutdown(context.Background()))
 	}()
 
 	factory := NewFactory()
@@ -68,9 +66,7 @@ func TestMetricsExporter(t *testing.T) {
 	tt, err := obsreporttest.SetupTelemetry()
 	require.NoError(t, err)
 	defer func() {
-		if err = tt.Shutdown(context.Background()); err != nil {
-			t.Fatalf("failed to stop: %v", err)
-		}
+		require.NoError(t, tt.Shutdown(context.Background()))
 	}()
 
 	factory := NewFactory()
@@ -95,9 +91,7 @@ func TestMetricsExporterSendError(t *testing.T) {
 	tt, err := obsreporttest.SetupTelemetry()
 	require.NoError(t, err)
 	defer func() {
-		if err = tt.Shutdown(context.Background()); err != nil {
-			t.Fatalf("failed to stop: %v", err)
-		}
+		require.NoError(t, tt.Shutdown(context.Background()))
 	}()
 
 	factory := NewFactory()

--- a/exporter/elasticexporter/internal/translator/elastic/exceptions_test.go
+++ b/exporter/elasticexporter/internal/translator/elastic/exceptions_test.go
@@ -300,10 +300,8 @@ func encodeSpanEvents(t *testing.T, language string, events ...ptrace.SpanEvent)
 	var recorder transporttest.RecorderTransport
 	resource := pcommon.NewResource()
 	resource.Attributes().InsertString(conventions.AttributeTelemetrySDKLanguage, language)
-	err := elastic.EncodeResourceMetadata(resource, &w)
-	assert.NoError(t, err)
-	err = elastic.EncodeSpan(span, pcommon.NewInstrumentationScope(), resource, &w)
-	assert.NoError(t, err)
+	assert.NoError(t, elastic.EncodeResourceMetadata(resource, &w))
+	assert.NoError(t, elastic.EncodeSpan(span, pcommon.NewInstrumentationScope(), resource, &w))
 	sendStream(t, &w, &recorder)
 
 	payloads := recorder.Payloads()

--- a/exporter/elasticexporter/internal/translator/elastic/metadata.go
+++ b/exporter/elasticexporter/internal/translator/elastic/metadata.go
@@ -98,21 +98,18 @@ func EncodeResourceMetadata(resource pcommon.Resource, w *fastjson.Writer) (err 
 
 	w.RawString(`{"metadata":{`)
 	w.RawString(`"service":`)
-	err = service.MarshalFastJSON(w)
-	if err != nil {
+	if err := service.MarshalFastJSON(w); err != nil {
 		return err
 	}
 	if system != (model.System{}) {
 		w.RawString(`,"system":`)
-		err = system.MarshalFastJSON(w)
-		if err != nil {
+		if err := system.MarshalFastJSON(w); err != nil {
 			return err
 		}
 	}
 	if len(labels) > 0 {
 		w.RawString(`,"labels":`)
-		err = labels.MarshalFastJSON(w)
-		if err != nil {
+		if err := labels.MarshalFastJSON(w); err != nil {
 			return err
 		}
 	}

--- a/exporter/elasticexporter/internal/translator/elastic/metadata_test.go
+++ b/exporter/elasticexporter/internal/translator/elastic/metadata_test.go
@@ -147,8 +147,7 @@ func metadataWithResource(t *testing.T, resource pcommon.Resource) metadata {
 	var out metadata
 	var recorder transporttest.RecorderTransport
 	var w fastjson.Writer
-	err := elastic.EncodeResourceMetadata(resource, &w)
-	assert.NoError(t, err)
+	assert.NoError(t, elastic.EncodeResourceMetadata(resource, &w))
 	sendStream(t, &w, &recorder)
 	out.system, out.process, out.service, out.labels = recorder.Metadata()
 	return out

--- a/exporter/elasticexporter/internal/translator/elastic/metrics_test.go
+++ b/exporter/elasticexporter/internal/translator/elastic/metrics_test.go
@@ -32,8 +32,7 @@ import (
 func TestEncodeMetrics(t *testing.T) {
 	var w fastjson.Writer
 	var recorder transporttest.RecorderTransport
-	err := elastic.EncodeResourceMetadata(pcommon.NewResource(), &w)
-	assert.NoError(t, err)
+	assert.NoError(t, elastic.EncodeResourceMetadata(pcommon.NewResource(), &w))
 
 	scopeMetrics := pmetric.NewScopeMetrics()
 	metrics := scopeMetrics.Metrics()

--- a/exporter/elasticexporter/internal/translator/elastic/traces_test.go
+++ b/exporter/elasticexporter/internal/translator/elastic/traces_test.go
@@ -161,8 +161,7 @@ func TestEncodeSpanStatus(t *testing.T) {
 			span.Status().SetCode(statusCode)
 		}
 
-		err = elastic.EncodeSpan(span, pcommon.NewInstrumentationScope(), pcommon.NewResource(), &w)
-		require.NoError(t, err)
+		require.NoError(t, elastic.EncodeSpan(span, pcommon.NewInstrumentationScope(), pcommon.NewResource(), &w))
 		sendStream(t, &w, &recorder)
 		payloads := recorder.Payloads()
 		require.Len(t, payloads.Transactions, 1)
@@ -520,10 +519,8 @@ func transactionWithAttributes(t *testing.T, attrs map[string]interface{}) model
 	pcommon.NewMapFromRaw(attrs).CopyTo(span.Attributes())
 
 	resource := pcommon.NewResource()
-	err := elastic.EncodeResourceMetadata(resource, &w)
-	assert.NoError(t, err)
-	err = elastic.EncodeSpan(span, pcommon.NewInstrumentationScope(), resource, &w)
-	assert.NoError(t, err)
+	assert.NoError(t, elastic.EncodeResourceMetadata(resource, &w))
+	assert.NoError(t, elastic.EncodeSpan(span, pcommon.NewInstrumentationScope(), resource, &w))
 	sendStream(t, &w, &recorder)
 
 	payloads := recorder.Payloads()
@@ -540,10 +537,8 @@ func spanWithAttributes(t *testing.T, attrs map[string]interface{}) model.Span {
 	pcommon.NewMapFromRaw(attrs).CopyTo(span.Attributes())
 
 	resource := pcommon.NewResource()
-	err := elastic.EncodeResourceMetadata(resource, &w)
-	assert.NoError(t, err)
-	err = elastic.EncodeSpan(span, pcommon.NewInstrumentationScope(), resource, &w)
-	assert.NoError(t, err)
+	assert.NoError(t, elastic.EncodeResourceMetadata(resource, &w))
+	assert.NoError(t, elastic.EncodeSpan(span, pcommon.NewInstrumentationScope(), resource, &w))
 	sendStream(t, &w, &recorder)
 
 	payloads := recorder.Payloads()

--- a/exporter/elasticexporter/internal/translator/elastic/traces_test.go
+++ b/exporter/elasticexporter/internal/translator/elastic/traces_test.go
@@ -33,8 +33,7 @@ import (
 func TestEncodeSpan(t *testing.T) {
 	var w fastjson.Writer
 	var recorder transporttest.RecorderTransport
-	err := elastic.EncodeResourceMetadata(pcommon.NewResource(), &w)
-	assert.NoError(t, err)
+	assert.NoError(t, elastic.EncodeResourceMetadata(pcommon.NewResource(), &w))
 
 	traceID := model.TraceID{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16}
 	rootTransactionID := model.SpanID{1, 1, 1, 1, 1, 1, 1, 1}
@@ -152,8 +151,7 @@ func TestEncodeSpanStatus(t *testing.T) {
 
 		var w fastjson.Writer
 		var recorder transporttest.RecorderTransport
-		err := elastic.EncodeResourceMetadata(pcommon.NewResource(), &w)
-		assert.NoError(t, err)
+		assert.NoError(t, elastic.EncodeResourceMetadata(pcommon.NewResource(), &w))
 		span := ptrace.NewSpan()
 		span.SetTraceID(pcommon.NewTraceID([16]byte{1}))
 		span.SetSpanID(pcommon.NewSpanID([8]byte{1}))
@@ -184,10 +182,8 @@ func TestEncodeSpanTruncation(t *testing.T) {
 
 	var w fastjson.Writer
 	var recorder transporttest.RecorderTransport
-	err := elastic.EncodeResourceMetadata(pcommon.NewResource(), &w)
-	assert.NoError(t, err)
-	err = elastic.EncodeSpan(span, pcommon.NewInstrumentationScope(), pcommon.NewResource(), &w)
-	require.NoError(t, err)
+	assert.NoError(t, elastic.EncodeResourceMetadata(pcommon.NewResource(), &w))
+	require.NoError(t, elastic.EncodeSpan(span, pcommon.NewInstrumentationScope(), pcommon.NewResource(), &w))
 	sendStream(t, &w, &recorder)
 
 	payloads := recorder.Payloads()
@@ -500,10 +496,8 @@ func TestInstrumentationLibrary(t *testing.T) {
 	library.SetVersion("1.2.3")
 
 	resource := pcommon.NewResource()
-	err := elastic.EncodeResourceMetadata(resource, &w)
-	assert.NoError(t, err)
-	err = elastic.EncodeSpan(span, library, resource, &w)
-	assert.NoError(t, err)
+	assert.NoError(t, elastic.EncodeResourceMetadata(resource, &w))
+	assert.NoError(t, elastic.EncodeSpan(span, library, resource, &w))
 	sendStream(t, &w, &recorder)
 
 	payloads := recorder.Payloads()

